bedrock/firefox/templates/firefox/features/send-tabs.html

```diff
@@ -35,10 +35,56 @@
 
 {% block features_header_content %}
   <h1><span>{{ _('Take your tabs to go') }}</span></h1>
-  <p class="tagline">{{ _('Share your open tabs seamlessly across mobile, desktop or tablet with Firefox.') }}</p>
+  {% if l10n_has_tag('send-tabs-update-08-2017') %}
+    <p class="tagline show-not-fx show-fx-31-signed-out show-fx-30-older show-default">{{ _('Share your open tabs seamlessly across mobile, desktop or tablet with Firefox.') }}</p>
+    <p class="tagline show-fx-31-signed-in">{{ _('View your open tabs across mobile and tablet devices by signing in to your Firefox Account.') }}</p>
+    <p class="tagline show-fx-android show-fx-ios">{{ _('View your open tabs across desktop and tablet too. Just sign in or create a Firefox account.') }}</p>
```

Comment:
It could work, but the string for signed-out is also the default, which covers a wider range of devices. I'm going to side against mixing this up again.